course/tests/tests_exportcourse.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management import CommandError, find_commands
+from django.test import TestCase
+from django.core import management
+from io import StringIO
+import sys
+
+
+class CommandTests(TestCase):
+    databases = '__all__'
+    fixtures = ["dumps/courseData.json"]
+
+    def test_command(self):
```

Comment:
this could be called `test_exportcourse` to be more specific

Reply:
Okay